### PR TITLE
Implemented nested template variables functionality for Opentsdb

### PR DESCRIPTION
**Pull request by utkarshcmu**
Resolving #4398

### CODE REVIEW

File: public/app/plugins/datasource/opentsdb/datasource.js [hunk 1]
```diff
@@ -162,12 +162,21 @@ function (angular, _, dateMath) {
...
         return $q.when([]);
       }
 
-      var m = metric + "{" + key + "=*}";
+      var keysArray = keys.split(",");
+      var key = keysArray[0];
+      var keysQuery = key + "=*";
+
+      if (keysArray.length > 1) {
+        keysQuery += "," + keysArray.splice(1).join(",");
```
Comment (gt-rocker) [inline public/app/plugins/datasource/opentsdb/datasource.js]: what about trimming spaces? OpenTSDB doesn't allow leading or trailing spaces in this part of query

Comment (utkarshcmu) [inline public/app/plugins/datasource/opentsdb/datasource.js]: Good point, I missed the corner case for having heading/trailing spaces. Will path the fix soon.

**Comment by utkarshcmu** [comment]
@gt-rocker Fixed the whitespace trimming issue. Thanks for the review.

@torkelo / @bergquist Please review/merge.

**Comment by bergquist** [comment]
Great work @utkarshcmu

**Comment by utkarshcmu** [comment]
@bergquist Thanks for reviewing/merging this PR.

**Pull request merged**